src/main/java/com/google/api/codegen/transformer/InitCodeTransformer.java

```diff
@@ -258,6 +264,25 @@ private InitCodeView buildInitCodeViewRequestObject(
     return paths.stream().map(path -> root.subTree(path)).collect(ImmutableList.toImmutableList());
   }
 
+  @VisibleForTesting
+  static void assertNoOverlap(InitCodeNode root, List<String> paths) {
```

Comment:
Add a function comment explaining what this does

Reply:
Done